<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Systemus::StandardUserInterface</class>
 <widget class="QWidget" name="Systemus::StandardUserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="searchInput">
       <property name="placeholderText">
        <string>Search...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="icon">
        <iconset resource="../resources/systemus_widgets_res.qrc">
         <normaloff>:/systemus/icons/action_refresh.png</normaloff>:/systemus/icons/action_refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addButton">
       <property name="icon">
        <iconset resource="../resources/systemus_widgets_res.qrc">
         <normaloff>:/systemus/icons/action_add.png</normaloff>:/systemus/icons/action_add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editButton">
       <property name="icon">
        <iconset resource="../resources/systemus_widgets_res.qrc">
         <normaloff>:/systemus/icons/action_edit.png</normaloff>:/systemus/icons/action_edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteButton">
       <property name="icon">
        <iconset resource="../resources/systemus_widgets_res.qrc">
         <normaloff>:/systemus/icons/action_delete.png</normaloff>:/systemus/icons/action_delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="printButton">
       <property name="icon">
        <iconset resource="../resources/systemus_widgets_res.qrc">
         <normaloff>:/systemus/icons/action_print.png</normaloff>:/systemus/icons/action_print.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="TableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSearchQuery">
   <property name="icon">
    <iconset resource="../resources/systemus_widgets_res.qrc">
     <normaloff>:/systemus/icons/search_icon.png</normaloff>:/systemus/icons/search_icon.png</iconset>
   </property>
   <property name="text">
    <string>Query</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../resources/systemus_widgets_res.qrc">
     <normaloff>:/systemus/icons/search_go.png</normaloff>:/systemus/icons/search_go.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header location="global">SystemusWidgets/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/systemus_widgets_res.qrc"/>
 </resources>
 <connections/>
</ui>
